blockscout-ens/bens-logic/tests/migrations/20231219085631_push_mock_gnosis_data.sql

```diff
@@ -444,7 +444,7 @@ INSERT INTO sgd2.domain (vid, block_range, id, name, label_name, labelhash, pare
 INSERT INTO sgd2.domain (vid, block_range, id, name, label_name, labelhash, parent, subdomain_count, resolved_address, resolver, ttl, is_migrated, created_at, owner, registrant, wrapped_owner, expiry_date) VALUES (253, '[7419183,)', '0x634ae5e4e77ee5a262a820f4a9eacd51ac137dd75989e5a5d993f5b1db797fba', '[959002e6c39d7f4735b5d874e0b6532d0b63a6404fe8b0978a5b357455421cca].[000027d83d9f5da7a98672efb882948cc95183501c1ae26ee8d49635711757d2]', NULL, '\x959002e6c39d7f4735b5d874e0b6532d0b63a6404fe8b0978a5b357455421cca', '0x1a13b687a5ff1d8ab1a9e189e1507a6abe834a9296cc8cff937905e3dee0c4f6', 64, NULL, NULL, NULL, true, 1700725990, '0xfd3d666db2557983f3f04d61f90e35cc696f6d60', NULL, NULL, NULL);
 INSERT INTO sgd2.domain (vid, block_range, id, name, label_name, labelhash, parent, subdomain_count, resolved_address, resolver, ttl, is_migrated, created_at, owner, registrant, wrapped_owner, expiry_date) VALUES (254, '[7419183,)', '0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2', '[e5e14487b78f85faa6e1808e89246cf57dd34831548ff2e6097380d98db2504a].[dec08c9dbbdd0890e300eb5062089b2d4b1c40e3673bbccb5423f7b37dcf9a9c]', NULL, '\xe5e14487b78f85faa6e1808e89246cf57dd34831548ff2e6097380d98db2504a', '0xa097f6721ce401e757d1223a763fef49b8b5f90bb18567ddb86fd205dff71d34', 57, NULL, NULL, NULL, true, 1700719455, '0xd322333e88acefc5bf19470d137c0f2272104ae8', NULL, NULL, NULL);
-INSERT INTO sgd2.domain (vid, block_range, id, name, label_name, labelhash, parent, subdomain_count, resolved_address, resolver, ttl, is_migrated, created_at, owner, registrant, wrapped_owner, expiry_date) VALUES (255, '[7419183,)', '0x6635ff2e6ba5d52ca5fa237e7e9f766bd73089f99ac49f49ba88ca61b8fbb168', 'zeshitcoinwebdev.gno', 'zeshitcoinwebdev', '\x1e371a39a036761bd34df0b795b63e9882bee59e71ce0be6210fe3546e097e3c', '0x634ae5e4e77ee5a262a820f4a9eacd51ac137dd75989e5a5d993f5b1db797fba', 0, '0xb1b38cedf091714916bc01bc4b96674b1bacdc10', '0x6d3b3f99177fb2a5de7f9e928a9bd807bf7b5bad-0x6635ff2e6ba5d52ca5fa237e7e9f766bd73089f99ac49f49ba88ca61b8fbb168', NULL, true, 1702945695, '0xb1b38cedf091714916bc01bc4b96674b1bacdc10', '0xb1b38cedf091714916bc01bc4b96674b1bacdc10', NULL, 1742278647);
+INSERT INTO sgd2.domain (vid, block_range, id, name, label_name, labelhash, parent, subdomain_count, resolved_address, resolver, ttl, is_migrated, created_at, owner, registrant, wrapped_owner, expiry_date) VALUES (255, '[7419183,)', '0xefc07af2d64eead3daec2e3004575bfc86bfc43c34e316294bd01c957e70cba2', 'abcnews.gno', 'abcnews', '\x1e371a39a036761bd34df0b795b63e9882bee59e71ce0be6210fe3546e097e3c', '0x1a13b687a5ff1d8ab1a9e189e1507a6abe834a9296cc8cff937905e3dee0c4f6', 0, '0xb1b38cedf091714916bc01bc4b96674b1bacdc10', '0x6d3b3f99177fb2a5de7f9e928a9bd807bf7b5bad-0x6635ff2e6ba5d52ca5fa237e7e9f766bd73089f99ac49f49ba88ca61b8fbb168', NULL, true, 1702945695, '0xb1b38cedf091714916bc01bc4b96674b1bacdc10', '0xb1b38cedf091714916bc01bc4b96674b1bacdc10', NULL, NULL);
```

Comment:
why did you changed parent hash `0x634ae5e4e77ee5a262a820f4a9eacd51ac137dd75989e5a5d993f5b1db797fba` to `0x1a13b687a5ff1d8ab1a9e189e1507a6abe834a9296cc8cff937905e3dee0c4f6` ?

`0x1a13b687a5ff1d8ab1a9e189e1507a6abe834a9296cc8cff937905e3dee0c4f6` is hash of empty label, `0x634ae5e4e77ee5a262a820f4a9eacd51ac137dd75989e5a5d993f5b1db797fba` hash of `.gno`.

parent of `abcnews.gno` is `gno`. almost all domains in this database have parent hash `0x634ae5e4e77ee5a262a820f4a9eacd51ac137dd75989e5a5d993f5b1db797fba`. it wont trigger any test for now, but it will be super hard to debug later